_includes/download/board.html

```diff
@@ -227,80 +227,86 @@ <h3>Previous Versions of CircuitPython</h3>
 {% endcomment %}
 
 {% if bootloader_version and bootloader_id %}
-{% if page.family == 'esp32s2' or page.family == 'esp32c3' or page.family == 'esp32s3' %}
+{% if page.family == 'esp32s2' or page.family == 'esp32s3' %}
 <div class="section unrecommended">
   <h3>Install, Repair, or Update UF2 Bootloader</h3>
   <p>
     Latest version: {{ bootloader_version }}
   </p>
   <p>
     <strong>The UF2 bootloader allows you to load CircuitPython, MakeCode, and Arduino programs.
-      The bootloader is not CircuitPython.</strong>
-    If a UF2 bootloader is installed, you can check its version by looking in
-    the <b>INFO_UF2.TXT</b> file when the <b>BOOT</b> drive is visible
-    (<b>FTHRS2BOOT</b>, <b>MAGTAGBOOT</b>, <b>HOUSEBOOT</b>, etc.)
+    </strong>
+    The bootloader itself is not CircuitPython.
+    On Espressif boards, the UF2 bootloader is called <em>TinyUF2</em>.
+    If the TinyUF2 bootloader is installed, you can check its version by looking in
+    the <strong>INFO_UF2.TXT</strong> file when the <strong>BOOT</strong> drive is visible
+    (<strong>FTHRS2BOOT</strong>, <strong>MAGTAGBOOT</strong>, <strong>HOUSEBOOT</strong>, etc.)
   </p>
   <p>
-    It is not necessary to reinstall a UF2 bootloader you unless a <b>BOOT</b> drive is not visible
-    when in UF2 bootloader mode, or you know of a problem with your current UF2 bootloader.
+    In general, it is not in general necessary to update TinyUF2 at every version change.
```

Comment:
small typo here. Could be "In general, it is not in generally necessary"

or just take out "generally" all together since it starts with "in general,"

Reply:
Ah, and I realize after posting this that I mis-read it, and generally is wrong. 

But it could still be removed to avoid repetition if we want. I don't have a strong opinion either way now that I re-read it correctly.